tests_metricflow/snapshots/test_custom_granularity.py/SqlPlan/DuckDB/test_custom_offset_window__plan0_optimized.sql

```diff
@@ -0,0 +1,84 @@
+test_name: test_custom_offset_window
+test_filename: test_custom_granularity.py
+sql_engine: DuckDB
+---
+-- Compute Metrics via Expressions
+SELECT
+  metric_time__day
+  , bookings AS bookings_offset_one_martian_day
+FROM (
+  -- Join to Time Spine Dataset
+  -- Pass Only Elements: ['bookings', 'metric_time__day']
+  -- Aggregate Measures
+  -- Compute Metrics via Expressions
+  SELECT
+    subq_17.ds__day__lead AS metric_time__day
+    , SUM(subq_13.bookings) AS bookings
+  FROM (
+    -- Offset Base Granularity By Custom Granularity Period(s)
+    WITH cte_6 AS (
```

Comment:
@plypaul is there a way to get this CTE to the top level even though it was built in a subquery?